### DISCUSSION TITLE
Unable to mount istio access logs to efs

### DISCUSSION THREAD

**Discussion by ravi-rapyder** [istio/istio#53441] [Q&A]
### Is this the right place to submit this?

- [X] This is not a security vulnerability or a crashing bug
- [X] This is not a question about how to use Istio

### Bug Description

Is there any way i can mount my istio access logs to efs - Lret me know

I have enabled access logs by following official documentation and now to mount it to efs how it is possible as you know 

/dev/stdout is an special file system and cannot mount it directly and regarding other files we are getting permission denied error read only file system error is there any way that we can do this ?

### Version

```Text
istio-1.22.1
```

**Comment by howardjohn** [comment]
Istio doesn't only write to /dev/stdout, you can configure the accessLogFile. You can also use https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#customizing-injection to inject a custom volume if that is required -- not sure how the efs setup works.
